pkg/collector/python/init.go

```diff
@@ -315,6 +315,12 @@ func Initialize(paths ...string) error {
 		C.add_python_path(rtloader, TrackedCString(p))
 	}
 
+	// On Windows, it's not uncommon to have a system-wide PYTHONPATH env var set.
+	// Remove it, so our embedded python doesn't try to load things from the system.
+	if runtime.GOOS == "windows" && !config.Datadog.GetBool("windows_use_pythonpath") {
```

Comment:
I would put this here instead: https://github.com/DataDog/datadog-agent/blob/albertvaka/unset-pythonpath/pkg/collector/python/init_windows.go#L12

Reply:
makes sense 👍